doc/specs/vulkan/chapters/devsandqueues.txt

```diff
@@ -347,6 +347,13 @@ include::../structs/VkDeviceCreateInfo.txt[]
 
 include::../validity/structs/VkDeviceCreateInfo.txt[]
 
+fname:vkCreateDevice creates a VkDevice object, then enables
+extensions requested by the application. If an extension
+is provided by a layer, the layer must: be specified at
```

Comment:
Oh... If I understood you correctly then that is a wrong usage of RFC "must" (for raising requirement purposes, **not** for stating truth).